### PR TITLE
Add Tag as input

### PR DESCRIPTION
**Pull request by stl-leanix**

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -4,6 +4,9 @@ inputs:
   namespace:
     description: 'kubernetes namespace'
     required: true
+  tag:
+    description: 'tag is required for Krane'
```
Comment (dan-leanix) [inline action.yml:8]:
```suggestion
    description: 'container image tag'
```

**Review by dan-leanix — CHANGES_REQUESTED** [review]
@stl-leanix Which story is behind that? I see no ticket number. If you have not created a ticket yet please do so and redo the changes on a new feature branch and close this PR.

File: action.yml [hunk 2]
```diff
@@ -4,6 +4,9 @@ inputs:
   namespace:
     description: 'kubernetes namespace'
     required: true
+  tag:
```
Comment (perbernhardt) [inline action.yml:7]: This is not true. A tag is only required, if your deployment templates ask for it. And you could choose for a completely different name there, e.g. "version".
See https://github.com/Shopify/krane#usage for required parameters.

File: action.yml (hunk 1 above)
Comment (stl-leanix) [reply action.yml:8]: Ok I will create a Ticket and redo! I only noticed this warning during our deployments and removed therefore the tag parameter and it crashed 😅 
Screenshot 2021-09-30 at 08 51 28

Comment (perbernhardt) [reply action.yml:8]: This is more like a "bug" in Github Actions: There is no way to express that you accept arbitrary inputs, although it is totally allowed to define and use them as a user.
Again: The "tag" variable is something you use inside of your templates, so you cannot remove it from the deployment...

**Pull request closed without merging**